experimental/PropertyDDS/examples/property-inspector/src/app.tsx

```diff
@@ -50,11 +50,6 @@ async function start(): Promise<void> {
     const propertyTree: IPropertyTree = await getDefaultObjectFromContainer<IPropertyTree>(container);
 
     renderApp(propertyTree.tree, document.getElementById("root")!);
-
-    // Reload the page on any further hash changes, e.g. in case you want to paste in a different document ID.
-    window.addEventListener("hashchange", () => {
-        location.reload();
-    });
```

Comment:
@karlbom, @evaliyev, @nedalhy - FYI.

Reply:
Oh yes, I should comment on this -- I was observing that this was triggering an overly aggressive reload which interrupted the initialization flow for the new document, causing the example to fail.  This bug already existed in main.